src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/Endpoint.kt

```diff
@@ -43,30 +46,28 @@ data class Endpoint(
 
     private fun addSecurityFilter(url: String)
     {
-        if (AppConfig().authEnabled)
```

Comment:
This `authEnabled` flag was being used to turn off all auth (i.e. authentication as well as authorization.) I've re-purposed it to just apply to authorization.

Reply:
So authEnabled can be configured to false if we don't want to bother with perms at all, and all users should be able to access all content? Could we rename to authorizationEnabled just to avoid confusion?